server/src/index.ts

```diff
@@ -124,98 +125,6 @@ app.get(
   },
 );
 
-app.get(
-  "/api/getContent/:userId",
-  async (req: Request, res: Response, next: NextFunction) => {
-    const loggedInUserId = Number(req.cookies.userId);
-    const userId = Number(req.params.userId);
-    try {
-      const contentData = await getFolderContent({
-        ownerId: userId,
-        loggedInUserId,
-        folderId: null,
-      });
-      const allDoenetmlVersions = await getAllDoenetmlVersions();
-      res.send({ allDoenetmlVersions, ...contentData });
-    } catch (e) {
-      if (e instanceof Prisma.PrismaClientKnownRequestError) {
-        res.status(404).send("No content found");
-      } else {
-        next(e);
-      }
-    }
-  },
-);
-
-app.get(
-  "/api/getContent/:userId/:folderId",
-  async (req: Request, res: Response, next: NextFunction) => {
-    const loggedInUserId = Number(req.cookies.userId);
-    const folderId = Number(req.params.folderId);
-    const userId = Number(req.params.userId);
-    try {
-      const contentData = await getFolderContent({
-        ownerId: userId,
-        loggedInUserId,
-        folderId,
-      });
-      const allDoenetmlVersions = await getAllDoenetmlVersions();
-      res.send({ allDoenetmlVersions, ...contentData });
-    } catch (e) {
-      if (e instanceof Prisma.PrismaClientKnownRequestError) {
-        res.status(404).send("No content found");
-      } else {
-        next(e);
-      }
-    }
-  },
-);
-
-app.get(
```

Comment:
Keep the public content ones, as we'll need them to allow folks to access folders from community.

Reply:
Maybe rename them as `getPublicFolderContent`, since `getContent` is now renamed to `getFolderContent`.